libvast/fbs/data.fbs

```diff
@@ -28,8 +28,13 @@ table String {
   value: string;
 }
 
+struct PatternOptions {
```

Comment:
This has to be a table rather than a struct, otherwise it's not extensible.

Reply:
I turned `PatternOptions` into a `table` in https://github.com/tenzir/vast/pull/2964/commits/bcca2fa13973b5bf448def2460e9ffe7e5319715

Reply:
This looks like it's addressed; resolving.